test/cptree.hs

```diff
@@ -0,0 +1,28 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+import Turtle
+import Filesystem.Path.CurrentOS ()
+import System.IO.Temp (withSystemTempDirectory)
+import Control.Monad (unless)
+
+check :: String -> Bool-> IO ()
+check errorMessage successs = unless successs $ error errorMessage
+
+-- This test fails by hanging
```

Comment:
If the test fails by hanging then this should use `System.Timeout.timeout` to detect the hang